src/aiida/orm/nodes/process/calculation/calcjob.py

```diff
@@ -43,22 +42,8 @@ def _get_objects_to_hash(self) -> List[Any]:
         anyway in the computation of the hash would mean that the hash of the node would change as soon as the process
         has started and the input files have been written to the repository.
         """
-        from importlib import import_module
-
-        objects = [
-            import_module(self._node.__module__.split('.', 1)[0]).__version__,
-            {
-                key: val
-                for key, val in self._node.base.attributes.items()
-                if key not in self._node._hash_ignored_attributes and key not in self._node._updatable_attributes
-            },
-            self._node.computer.uuid if self._node.computer is not None else None,
-            {
-                entry.link_label: entry.node.base.caching.get_hash()
-                for entry in self._node.base.links.get_incoming(link_type=(LinkType.INPUT_CALC, LinkType.INPUT_WORK))
-                if entry.link_label not in self._hash_ignored_inputs
-            },
-        ]
+        objects = super().get_objects_to_hash()
```

Comment:
As far as I can see, there is one functional change here: class attribute is added from the parent class? I am assuming that is desired?

Reply:
You mean the `class` key with `str(self._node.__class__)` that I added in another PR yesterday? If so, yes, that is desired. The only exception for the `CalcJobNodeCaching` should be the removal of the repository hash, as added in #5998

Reply:
Yep, that's what I meant, all good then.